samples/msal-node-samples/auth-code-key-vault/index.js

```diff
@@ -4,7 +4,6 @@
  */
 
 const express = require('express');
-const forge = require('node-forge');
```

Comment:
Are we referencing `node-forge` only in the docs now?

Reply:
The code sample itself wasn't actually using it, but you remined me to add it to the snippet for pfx-pem conversion